scripts/zip-projects.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script automatically zips up our projects for upload to Github on release.
+# Invoke it like: ./zip-projects.sh <release_version> 
+# It will produce 3 zip files in your current directory which you can upload as a release.
+
+set -e -u -x -o pipefail
+
+function cleanUp() {
+    rm -rf "${TEMP_DIR}"
+}
+
+trap cleanUp EXIT
+
+RELEASE_VERSION=$1
+
+TEMP_DIR=$(mktemp -d || mktemp -d -t tmp)
+
+# Step 1. GDK for Unity repo.
+CORE_REPO="${TEMP_DIR}/core/gdk-for-unity"
+CORE_ZIP="$(pwd)/gdk-for-unity-${RELEASE_VERSION}.zip"
+git clone git@github.com:spatialos/gdk-for-unity.git "${CORE_REPO}"
+
+pushd "${CORE_REPO}"
+    tar cf "${CORE_ZIP}" *
+popd
+
+# Step 2. FPS 
+FPS_REPO="${TEMP_DIR}/fps/gdk-for-unity-fps-starter-project"
+FPS_ZIP="$(pwd)/gdk-for-unity-fps-starter-project-${RELEASE_VERSION}.zip"
+git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git "${FPS_REPO}"
+
+pushd "${FPS_REPO}"
+    ./ci/bootstrap.sh
+    # Get rid of heavy & unnecessary folders before zipping it. Can't get tar to ignore hidden subdirectories sadly.
+    rm -rf ./.git
+    rm -rf ./.shared-ci
```

Comment:
Probably don't need to zip `./scripts` either because we'll bundle gdk/fps together

Reply:
same applies to the blank project

Reply:
Similar comment as to above